ci_framework/roles/cifmw_cephadm/tasks/dashboard/validation.yml

```diff
@@ -0,0 +1,4 @@
+- name: Validate connection to dashboard service
+  ansible.builtin.command: "curl -k http://{{ grafana_server_addr }}:{{ cifmw_cephadm_dashboard_port }}"
+  register: dashboard_response
+  failed_when: dashboard_response.rc != 0
```

Comment:
Yes, this is the idea I had in mind. We can add more advanced curl tests in here just to make sure the dashboard is online. E.g. use the default password to login and look for correct 20x HTTP codes.

Reply:
Yeah, i added another test for checking the http code 20x, i will try to add more tests in followup commits.